### PR TITLE
docs: clarify how `bumpVersion` for `sbt` works

### PR DESCRIPTION
**Pull request by rgitzel**
## Changes:

No change in behaviour, just a small change to the docs.

## Context:

we put our version string in `project/Version.scala` instead of `build.sbt` for reasons. 

if updating PRs by hand becomes annoying enough, we'll see about adding something Renovate. 

thanks for this great tool! :-)

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Comment by HonkingGoose** [comment]
Please change the PR title to: 

docs: clarify how `bumpVersion` for `sbt` works

This way the correct Semantic Commit scope is used. 😉

**Comment by rgitzel** [comment]
> This way the correct Semantic Commit scope is used. 😉

Ah, not seen that kind of thing before.  Cool.

**Comment by HonkingGoose** [comment]
> > This way the correct Semantic Commit scope is used. 😉
> 
> Ah, not seen that kind of thing before. Cool.

Whoops, I said _Semantic Commit_, but we actually follow the _Conventional Commits_ specification. 🙈

If you're interested and want to know more, you can find the specification here: https://www.conventionalcommits.org/en/v1.0.0/

The Renovate maintainers with write rights can always edit the PR title, so it's no big problem. 😄

File: docs/usage/configuration-options.md [hunk 1]
```diff
@@ -283,6 +283,8 @@ For `npm` only you can also configure this field to `"mirror:x"` where `x` is th
 Doing so means that the `package.json` `version` field will mirror whatever the version is that `x` depended on.
 Make sure that version is a pinned version of course, as otherwise it won't be valid.
 
+For `sbt` note that Renovate will update the version string only if it's found in the project's `built.sbt` file.
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:286]:
```suggestion
For `sbt`, note that Renovate can only update packages that have a version string in their project's `built.sbt` file.
```

I find the original sentence hard to understand. So how about something like this?

I don't know if my suggestion is technically-correct, so please check committing it. 😉

Comment (rgitzel) [reply docs/usage/configuration-options.md:286]: I was following the example in line 182.  And the comma isn't actually necessary: we make a pause when we say it outloud, but we don't write it down.  "For sbt" isn't a subordinate clause.

"can only update packages" makes it sound like Renovate won't do _any_ updates if the version string isn't in that file.  

I think my wording is more correct, in that it is specific to the version string being updated.

How about?
```
For `sbt` note that Renovate will update the version string only for packages that have the version string in their project's `built.sbt` file.
```

Comment (HonkingGoose) [reply docs/usage/configuration-options.md:286]: I like your new text way better! ✨ 

But put it on one line. We use one line per sentence in our docs. This makes it easier to work with the documentation, as you can move lines around easily. 😉 

```suggestion
For `sbt` note that Renovate will update the version string only for packages that have the version string in their project's `built.sbt` file.
```

---

> I was following the example in line 182. And the comma isn't actually necessary: we make a pause when we say it outloud, but we don't write it down. "For sbt" isn't a subordinate clause.

I do tend to put too many commas in my text, so I'll take your advice on this. 😉

**Review by HonkingGoose — APPROVED** [review]

**Review by rarkins — APPROVED** [review]

**Pull request merged**